index.html

```diff
@@ -87,20 +84,20 @@ <h1 class="my-4 text-5xl font-bold leading-tight">
           </p>
           <a href="https://github.com/web3community/start-here" target="_blank" rel="noopener">
             <button style="background: #0086D6" class="mx-auto lg:mx-0 hover:underline text-white rounded-md my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
-              Explore
+              Explore Now
```

Comment:
I think `Explore` sounds better. This doesn't have to be accepted, just a suggestion. 🙂
```suggestion
              Explore
```

Reply:
I'm ok with your suggestion, think same. :)